tests/test_cmd2.py

```diff
@@ -1761,6 +1761,15 @@ def test_alias(base_app, capsys):
     out = run_cmd(base_app, 'alias fake')
     assert out == normalize('alias fake pyscript')
 
+def test_alias_with_quotes(base_app, capsys):
```

Comment:
I recommend creating another unit test where the thing quoted isn’t a redirector.  Keep this one too.

Reply:
2 things are quoted in the unit test
alias fake help ">" "out file.txt"

Reply:
Oops didn’t see that.  Thanks for pointing it out.